include/gsl/gsl

```diff
@@ -19,32 +19,28 @@
 #ifndef GSL_GSL_H
 #define GSL_GSL_H
 
-#include "gsl_assert"  // Ensures/Expects
-#include "gsl_util"    // finally()/narrow()/narrow_cast()...
-#include "multi_span"  // multi_span, strided_span...
-#include "span"        // span
-#include "string_span" // zstring, string_span, zstring_builder...
+#include <gsl/gsl_assert>  // Ensures/Expects
+#include <gsl/gsl_util>    // finally()/narrow()/narrow_cast()...
+#include <gsl/multi_span>  // multi_span, strided_span...
+#include <gsl/span>        // span
+#include <gsl/string_span> // zstring, string_span, zstring_builder...
 #include <memory>
 
-#ifdef _MSC_VER
-
-// No MSVC does constexpr fully yet
-#pragma push_macro("constexpr")
-#define constexpr /*constexpr*/
-
-// MSVC 2013 workarounds
-#if _MSC_VER <= 1800
-// noexcept is not understood
-#pragma push_macro("noexcept")
-#define noexcept /*noexcept*/
+#if defined(_MSC_VER) && _MSC_VER < 1910
+ #pragma push_macro("constexpr")
+ #define constexpr inline
 
```

Comment:
I would rather we don't to that define, in case we need a constexpr variable at some point.